### PR TITLE
documentation for #478 and some more cleanup of ltnews33

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Feedback wanted  (any corrections, typos, ...)
- Ready to merge

## Checklist of required changes before merge will be approved
- [n/a] Test file(s) added
- [n/a] Version and date string updated in changed source files
- [n/a] Relevant `\changes` entries in source included
- [n/a ] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` updated

### CODE REVIEW

File: base/doc/ltnews33.tex [hunk 1]
```diff
@@ -358,23 +512,20 @@ \subsection{Execute \cs{par} at the end of \cs{marginpar} arguments}
...
-It is sometimes necessary to reference the same footnote several
-times, i.e., produce several footnote marks with the same number or
-symbol. This is now always possible by placing a \cs{label} into the
-\cs{footnote} and reference it with the command \cs{footref}
-elsewhere.  This way marks refering to footnotes anywhere on the page
-(including those in \texttt{minipage}s) can be generated.  In the past
-this command was only available with certain classes or when loading
-the \pkg{footmisc} package.
+With the NFSS extensions introduced in 2020 the font series and shape
+settings be be influenced by changes to the font family. The setting
```
Comment (mrpiggi) [inline base/doc/ltnews33.tex:518]: "be" doubled

Comment (FrankMittelbach) [reply base/doc/ltnews33.tex]: five others elsewhere as well, thanks ... I should always use Nelson's "dw" command for checking :-(

File: base/doc/ltnews33.tex [hunk 2]
```diff
@@ -135,17 +135,156 @@ \section{Extending the hook concept to paragraphs}
...
+them accessible. Unfortunately the \TeX{} community was a bit too
+enthusiastic and included several symbols only available in a few
+\TeX{} fonts and some, such as the capital accents, not available at
+all but developed as part of the reference font implementation.
+
+In hindsight that was a very bad idea because it meant that other
+existing fonts (at the time) and later new fonts that got developed
+were unable to provide the full set of glyphs that made up the
+\texttt{TS1} encoding. For existing free PostScript fonts people to
+the extra effort and produced virtual fonts that faked (some) of
```
Comment (davidcarlisle) [inline base/doc/ltnews33.tex:191]: took

File: base/doc/ltnews33.tex [hunk 3]
```diff
@@ -135,17 +135,156 @@ \section{Extending the hook concept to paragraphs}
...
+
+At the time this happened many new free fonts had appeared and
+unfortunately the chaos around the question \enquote{which glyphs of
+  the \texttt{TS1} encoding are implemented by which font} had
+increased with it. Not only did one find many new holes it was next to
+impossible to order the set of fonts into a reasonable set of
+sub-encodings that are contained in each other in a single sequence.
+
+In the end we decided on nine or ten sub-encodings with a reasonable
+number of font in each so that all font implemented all glyphs of the
```
Comment (davidcarlisle) [inline base/doc/ltnews33.tex:212]: so that all fonts

File: base/doc/ltnews33.tex [hunk 4]
```diff
@@ -135,17 +135,156 @@ \section{Extending the hook concept to paragraphs}
...
 
-\subsection{Adjusting \env{itemize} labels with \cs{labelitemfont}}
+
+\subsection{Support several footnote marks to one footnote}
+
+It is sometimes necessary to reference the same footnote several
+times, i.e., produce several footnote marks with the same number or
+symbol. This is now always possible by placing a \cs{label} into the
+\cs{footnote} and reference it with the command \cs{footref}
+elsewhere.  This way marks referring to footnotes anywhere on the page
```
Comment (davidcarlisle) [inline base/doc/ltnews33.tex:260]: In this way,   marks ...

**Review by davidcarlisle — COMMENTED** [review]
I noted a couple of minor wording changes inline.

**Comment by car222222** [comment]
Thanks for these, but two of them are in a section (about fonts and TS1 etc.) that does not appear to be any longer part of this document.  

Did it get lost somewhere?

**Comment by FrankMittelbach** [comment]
> Thanks for these, but two of them are in a section (about fonts and TS1 etc.) that does not appear to be any longer part of this document.
> 
> Did it get lost somewhere?

yes, looks like the documentation for #478 never got merged back into develop. I'll take a look and see if I can merge it into the current file.

**Comment by FrankMittelbach** [comment]
I now moved the relevant section of 478 into ltnew33 (but haven't addressed any comments by David from above).

**Comment by FrankMittelbach** [comment]
typos handled and integration done elsewhere

**Pull request closed without merging**